tcms/core/tests/test_ajax.py

```diff
@@ -71,123 +72,43 @@ def test_update_default_tester_via_email(self):
         cls.url = reverse('ajax-tags')
         cls.test_tag = TagFactory()
         cls.test_plan = TestPlanFactory()
         cls.test_case = TestCaseFactory()
         cls.test_run = TestRunFactory()
 
-
-class Test_Tag_Add(Test_Tag_Test):
-
-    def test_add_tag_to_test_plan(self):
-        response = self.client.get(self.url, {
-            'tags': self.test_tag,
-            'plan': self.test_plan.plan_id,
-            'a': 'add'
-        })
-
-        self.assertEqual(response.status_code, HTTPStatus.OK)
-        self.assertEqual(self.test_plan.tag.count(), 1)
-        self.assertTrue(self.test_tag in self.test_plan.tag.all())
-
-    def test_add_tag_to_test_case(self):
-        response = self.client.get(self.url, {
-            'tags': self.test_tag,
-            'case': self.test_case.case_id,
-            'a': 'add'
-        })
-
-        self.assertEqual(response.status_code, HTTPStatus.OK)
-        self.assertEqual(self.test_case.tag.count(), 1)
-        self.assertTrue(self.test_tag in self.test_case.tag.all())
-
-    def test_add_tag_to_test_run(self):
-        response = self.client.get(self.url, {
-            'tags': self.test_tag,
-            'run': self.test_run.run_id,
-            'a': 'add'
-        })
-
-        self.assertEqual(response.status_code, HTTPStatus.OK)
-        self.assertEqual(self.test_run.tag.count(), 1)
-        self.assertTrue(self.test_tag in self.test_run.tag.all())
-
-
-class Test_Tag_Remove(Test_Tag_Test):
-
-    def test_remove_tag_from_test_plan(self):
-        self.test_plan.add_tag(self.test_tag)
-
-        response = self.client.get(self.url, {
-            'tags': self.test_tag,
-            'plan': self.test_plan.plan_id,
-            'a': 'remove'
-        })
-
-        self.assertEqual(response.status_code, HTTPStatus.OK)
-        self.assertEqual(self.test_plan.tag.count(), 0)
-
-    def test_remove_tag_from_test_case(self):
-        self.test_case.add_tag(self.test_tag)
-
-        response = self.client.get(self.url, {
-            'tags': self.test_tag,
-            'case': self.test_case.case_id,
-            'a': 'remove'
-        })
-
-        self.assertEqual(response.status_code, HTTPStatus.OK)
-        self.assertEqual(self.test_case.tag.count(), 0)
-
-    def test_remove_tag_from_test_run(self):
-        self.test_run.add_tag(self.test_tag)
-
-        response = self.client.get(self.url, {
-            'tags': self.test_tag,
-            'run': self.test_run.run_id,
-            'a': 'remove'
-        })
-
-        self.assertEqual(response.status_code, HTTPStatus.OK)
-        self.assertEqual(self.test_run.tag.count(), 0)
-
-
-class Test_Tag_Render(Test_Tag_Test):
-
-    @classmethod
-    def setUpTestData(cls):
-        super(Test_Tag_Render, cls).setUpTestData()
-
         cls.test_plan.add_tag(cls.test_tag)
         cls.test_case.add_tag(cls.test_tag)
         cls.test_run.add_tag(cls.test_tag)
 
-        for _ in range(0, 3):
+        for i in range(0, 3):
```

Comment:
these are unnecessary and cause pylint to report "tcms/core/tests/test_ajax.py:105:12: W0612: Unused variable 'i' (unused-variable)". I will merge this as it is but fix pylint issues reported for these new test files right afterwards.